stories/components/context-menu.stories.tsx

```diff
@@ -15,6 +14,15 @@ export default {
   title: 'Components / ContextMenu',
 };
 
+const ContextMenuStyled = styled(ContextMenu)`
```

Comment:
It sound more like english if you put the adjective in the front => `StyledContextMenu`